signer/src/storage/model.rs

```diff
@@ -557,16 +553,19 @@ impl From<SignerVotes> for BitArray<[u8; 16]> {
     }
 }
 
-/// Possible states for `dkg_shares` entries.
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord, strum::Display)]
+/// The possible states for DKG shares.
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord, sqlx::Type)]
+#[sqlx(type_name = "dkg_shares_status", rename_all = "snake_case")]
 #[cfg_attr(feature = "testing", derive(fake::Dummy))]
+// #[strum(serialize_all = "snake_case")]
```

Comment:
nit: leftover `strum`

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1316/commits/1fb3ffa10c36b8349fddcd3ad0b08aeee63bf362